yt_dlp/extractor/youtube.py

```diff
@@ -5379,10 +5418,19 @@ def get_mobj(url):
         tabs = traverse_obj(data, ('contents', 'twoColumnBrowseResultsRenderer', 'tabs'), expected_type=list)
         if tabs:
             selected_tab = self._extract_selected_tab(tabs)
+            selected_tab_url = urljoin(
+                url, traverse_obj(selected_tab, ('endpoint', 'commandMetadata', 'webCommandMetadata', 'url')))
+
             selected_tab_name = selected_tab.get('title', '').lower()
+
+            # Prefer tab name from tab url as it is always in en
+            if selected_tab_url:
+                selected_tab_name = get_mobj(selected_tab_url)['tab'][1:] or selected_tab_name
```

Comment:
There are so many types of pages that it'll be difficult to fully test this 😒

Reply:
If it proves to be too difficult we could change the preference based on if preferred_lang is set